### ISSUE TITLE
images of trivial group

### ISSUE THREAD

**Issue by jdbancal** [replab/replab#146]
When constructing a group with no generator, the number of generator is not zero but 1. Then, when asking for the definingRep, the image of a generator is constructed, which is empty, so fails to be of dimension `dxd`:

```
g = replab.Permutations(3).subgroup({[]})
g.definingRep
```
fails. Same happens with
```
g = replab.Permutations(3).subgroup({zeros(0,3)})
g.definingRep
```

**Comment by denisrosset** [comment]
@jdbancal The example should be:
```
g = replab.Permutations(3).subgroup({})
g.naturalRep
```
i.e. it's an empty cell array, not a cell array containing an empty generator.
Nevertheless, there was a bug in `RepByImages` which is fixed by 9078c92fca65e7e2935bd310dce48cf1563285c5

**Comment by jdbancal** [comment]
Great, thanks!
